### DISCUSSION TITLE
What if I want to hide the scroll bar in various browsers?

### DISCUSSION THREAD

**Discussion by geekskai** [chakra-ui/chakra-ui#1963] [General] [answered]
https://next.chakra-ui.com/docs/features/style-props#pseudo
I checked the document carefully. If the page scrolls, you need to hide the scroll bar at this time. You need to use ::-webkit-scrollbar, but this pseudo-class can only be used in Chrome. If you need to be compatible with other current mainstream browsers, How to do it?

**Comment by with-heart** [comment, marked as answer]
Hi @GitHubGanKai! Since this is a more general CSS question, it's probably better suited to StackOverflow than Chakra Discussions. I found something that might be related for you though: https://stackoverflow.com/questions/16670931/hide-scroll-bar-but-while-still-being-able-to-scroll
